src/main/java/geekbrains/java2/ht2/Main.java

```diff
@@ -0,0 +1,42 @@
+package geekbrains.java2.ht2;
+
+public class Main {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.